osu.Game.Tests/Visual/Online/TestSceneOnlineViewContainer.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Online;
+using osu.Game.Online.API;
+using osuTK.Graphics;
+
+namespace osu.Game.Tests.Visual.Online
+{
+    [TestFixture]
+    public class TestSceneOnlineViewContainer : OsuTestScene
+    {
+        private OnlineViewContainer onlineView;
+        private Box box;
+
+        public TestSceneOnlineViewContainer()
+        {
+            Child = new Container
+            {
+                RelativeSizeAxes = Axes.Both,
+                Child = onlineView = new OnlineViewContainer(@"Please login to view dummy test content")
```

Comment:
Why not “sign in”?

Reply:
One thing I was aware from this is the placeholder message becoming inconsistent across the overlays by simply one of them having `log in` instead of `sign in`, I think the string should be concatenated somewhere inside the `LoginPlaceholder` in a way that the developer only need to pass reason to log in: 
```
OnlineViewContainer(@"dummy test content") -> $@"Please sign in to view {actionMessage}" -> "Please sign in to view dummy test content"
```

Reply:
This was in the initial implementation of the login placeholder but has been reverted to passing the whole string because string concatenation was deemed bad for localization.

Reply:
I mean, it will be concatenated before locallizing, at the constructor of `LoginPlaceholder`.

Reply:
Sounds fine in that case.